analyses/clone-phylogeny-analysis/05-run-numbat.R

```diff
@@ -51,7 +52,7 @@ df_allele_list <- list()
 
 #######################################################
 # Read metadata file and define `sample_name`
-metadata_file <- file.path(metadata_dir, "project_metadata.tsv") # metadata input file
+metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```

Comment:
Same commenting regarding the `metadata_file` variable here and on line `58`.

```suggestion
project_metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```

Reply:
Fixed here: 0f238c2d3a172b41d6d0fd3842f3f90d086f6874.